androidemu/native/memory_heap.py

```diff
@@ -50,6 +53,7 @@ def map(self, size, prot=UC_PROT_READ | UC_PROT_WRITE):
 
         # Actually map in emulator.
         self._uc.mem_map(data_addr, data_size, perms=prot)
+        logger.debug("mmap heap memory at 0x%x, size %d" % (data_addr, data_size))
```

Comment:
Can u remove this so I can merge

Reply:
OK, the commit related to this is reverted.